src/MVVM/ViewModels/Navigation/ViewModelBase.cs

```diff
@@ -58,7 +58,7 @@ await closeViewModelTask
         }
 
         /// <summary>
-        /// Closes this ViewModel and informs the parent using the IViewModelNavigationService (if registered)
+        /// Closes this ViewModel and informs the parent using the <see cref="CodeMonkeys.Navigation.INavigationService"/> (if registered)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 138 characters long line (which is greater than 130 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5674091)